plantuml-parser-core/src/main/java/com/shuzijun/plantumlparser/core/ParserConfig.java

```diff
@@ -33,6 +33,8 @@ public class ParserConfig {
 
     private boolean showConstructors = false;
 
+    private boolean showDefaultConstructors = false;
```

Comment:
You need to add a file analogous to [ShowConstructorsOption.java](https://github.com/cathylee12/plantuml-parser/blob/main/plantuml-parser-cli/src/main/java/com/shuzijun/plantumlparser/cli/options/showconstructors/ShowConstructorsOption.java). You should then manually test the new flag (boolean option) from the command line, including that it appears in help messages (presumably `-h`).

Reply:
Could you please explain more about this?